StreamChatClient_v3/ChatClient.swift

```diff
@@ -233,3 +242,18 @@ extension ClientError {
         public let localizedDescription: String = "The URL provided in ChatClientConfig is `nil`."
     }
 }
+
+extension Client: ConnectionIdProvider {
+    func requestConnectionId(completion: @escaping (String?) -> Void) {
```

Comment:
Shouldn't the function name be `provideConnectionId`?

Reply:
Can be 👍  This is really just a temp solution. I'm working on a proper one in CIS-169